Fix:
../../tk/tkconfig.h:55:10: fatal error: 'X11/Xlib.h' file not found
#include <X11/Xlib.h>
         ^~~~~~~~~~~~
--- src/tclx/config/linux.orig	2018-06-29 04:19:31.000000000 -0500
+++ src/tclx/config/linux	2022-05-16 21:16:36.000000000 -0500
@@ -9,5 +9,6 @@
 RANLIB_CMD=ranlib
 MCS_CMD=true
 TCL_TK_LIBS= -L/usr/X11/lib -lX11 -lm -lXpm
+XINCLUDE=/usr/include/X11
 
 TCL_MAN_SEPARATOR=
--- src/tclx/config.mk.orig	2018-06-29 04:19:31.000000000 -0500
+++ src/tclx/config.mk	2022-05-16 21:16:30.000000000 -0500
@@ -54,6 +54,7 @@
 TCL_TK_DIR=../tk
 TK_LIBRARY=/usr/local/lib/tk
 XPM_LIBS=-L/usr/X11R6/lib -lXpm 
+XINCLUDE=/usr/X11R6/include
 
 #------------------------------------------------------------------------------
 # Compiler debug/optimization/profiling flag to use.  Not that if debugging or
--- src/tclx/tkucbsrc/makefile.orig	2018-06-29 04:19:31.000000000 -0500
+++ src/tclx/tkucbsrc/makefile	2022-05-16 21:18:25.000000000 -0500
@@ -24,7 +24,7 @@
 #------------------------------------------------------------------------------
 
 CFLAGS= $(OPTIMIZE_FLAG) $(XCFLAGS) -I../src -I../$(TCL_TK_DIR) \
-        -I../$(TCL_UCB_DIR) -I$(XHOME)/include \
+        -I../$(TCL_UCB_DIR) -I$(XINCLUDE) \
 	$(MEM_DEBUG_FLAGS) $(SYS_DEP_FLAGS) \
         -DTK_VERSION=\"2.2/\"
 
